modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -39,7 +44,12 @@ LABEL pxeboot
 This configuration does not enable serial console access on machines with a graphical console. To configure a different console, add one or more `console=` arguments to the `APPEND` line. For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console. For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
 ====
 
+ifndef::ibm-power[]
 ** For iPXE (`x86_64` + `aarch64`):
+endif::ibm-power[]
+ifdef::ibm-power[]
+** For iPXE (`x86_64` + `ppc64le`):
```

Comment:
are lines 47 to 49 suppose to use ibm-power since it state x86_64 and aarch64, i think it should have a different ifndef